crates/nu-protocol/src/parse_error.rs

```diff
@@ -561,7 +573,8 @@ impl ParseError {
             ParseError::ShellOutErrRedirect(s) => *s,
             ParseError::UnknownOperator(_, _, s) => *s,
             ParseError::InvalidLiteral(_, _, s) => *s,
-            ParseError::NotAConstant(s) => *s,
+            // ParseError::NotAConstant(s) => *s,
+            // ParseError::NotAConstCommand(s) => *s,
```

Comment:
could probably just take these (and the above) out. git will remember the history